### DISCUSSION TITLE
25 W battery discharge rate on eco mode

### DISCUSSION THREAD

**Discussion by monsterofweld** [seerge/g-helper#2632] [Q&A] [answered]
im getting really high discharge rate in eco mode on battery for some reason on low CPU settings and 60hz refresh rate.
is there something im missing and just overlooked?
any help would be appreciated

<img width="821" alt="Screenshot_24" src="https://github.com/seerge/g-helper/assets/170727835/2dfd8545-4a6a-44b6-8130-34f9bcc2b7d4">
<img width="819" alt="Screenshot_25" src="https://github.com/seerge/g-helper/assets/170727835/246ed36f-f827-4ec7-ab38-73cd3c9b40b0">
<img width="473" alt="Screenshot_26" src="https://github.com/seerge/g-helper/assets/170727835/6fb3ce14-597c-4981-8612-37119378e41b">

**Comment by seerge** [comment]
@monsterofweld try to enable and disable GPU again, and reboot.

G15 is a very strange device, with lots of issues. It sometimes fails to disable GPU properly, despite reporting that it did.

**Reply by monsterofweld** [reply]
ok so it kind of worked, after reboot i noticed slightly lower W usage but its still at around 10-12W on just browser operation. is that normal? ive disabled cpu boost and that seemed to help and lowered max cpu W output. (still in eco mode)

<img width="471" alt="Screenshot_30" src="https://github.com/seerge/g-helper/assets/170727835/680277d9-21dc-4499-95f0-c23b87e9e8e7">

**Reply by seerge** [reply, marked as answer]
Yes, 10W is ok.

From your screenshot CPU takes 8W + something for screen / fans / backlight.

Try to close all apps, set battery saver in windows and just wait for a few minutes. If power drain drops to something like 6-8W total, you are good.

**Reply by monsterofweld** [reply]
ok awesome ty.
